### DISCUSSION TITLE
Image resize is pixellating the resized image

### DISCUSSION THREAD

**Discussion by gmek5476** [ImageMagick/ImageMagick#6567] [Digital Image Processing]
We are trying to resize an image by using the below command
```
magick "C:\Im-Test\Input\Shop CMS Banner DTC Offer.png" -intent relative -sample 1500x1500> "-background white" -alpha remove -flatten -alpha off -quality 95 -density 96 -colorspace sRGB -strip -auto-orient C:\Im-Test\Output\cut-Out\result.png
```
After resizing we noticed that the result images is pixelated. Is there anyway we can get an image without pixelated when we resize.I have attached source and result images to this thread.

![result](https://github.com/ImageMagick/ImageMagick/assets/126016862/e4a76a9c-5362-4c20-b42a-82eed1b1e12f)
![Shop CMS Banner DTC Offer](https://github.com/ImageMagick/ImageMagick/assets/126016862/ed123d23-0435-4561-8b98-284440ec6061)

**Comment by snibgo** [comment]
I suggest using `-resize` instead of `-sample`.

**Comment by gmek5476** [comment]
Hi Snigbo thanks for yout support.
May i know what is the difference between -sample and -resize?
Iam not professionl in IM.

**Comment by fmw42** [comment]
See

https://imagemagick.org/script/command-line-options.php#sample

https://imagemagick.org/script/command-line-options.php#resize

**Comment by snibgo** [comment]
See also my recent reply to another thread https://github.com/ImageMagick/ImageMagick/discussions/6474#discussioncomment-6711561

**Comment by gmek5476** [comment]
Thanks Snibgo & Fred .Appreciate your quick help .This solution worked out for us.
